script/CIRCOS/final-plotting.R

```diff
@@ -21,6 +21,7 @@ parser$add_argument('--sample.name', type = 'character', help = 'Sample name', r
 parser$add_argument('--script.source', type = 'character', help = 'Path to directory containing helper scripts', required = TRUE);
 parser$add_argument('--sv.caller', type = 'character', help = 'Either Delly, Manta, or SURVIVOR verbatim', default = 'Delly');
 parser$add_argument('--plot.title', type = 'logical', help = 'print a title? TRUE or FALSE', default = TRUE);
+parser$add_argument('--genome.build', type = 'character', help = 'Genome build (e.g. hg38, hg19)', default = 'hg38');

```

Comment:
question:

Is this correct? I guess this process doesn't work with the mouse genome, which is fine for now, but does this fail with a mouse sample?

```suggestion
parser$add_argument('--genome.build', type = 'character', help = 'Genome build (hg38 or hg19)', default = 'hg38');

```

Reply:
I haven't tested with a mouse sample, I believe CIRCOS does support the mouse genome but the process on Nextflow's side hasn't been set up to allow it in this PR until proper testing with a mouse sample

Reply:
It looks like it supports the mouse genome. Yup, separate PR, for sure. We can create an issue to track it.

https://circos.ca/tutorials/lessons/configuration/data_files/#:~:text=karyotype%20%E2%80%94%20biology%20applications,all%20chromosomes%20will%20be%20drawn.&text=chr%20%2D%20hs1%201%200%20249250621,karyotype%20tutorial%20for%20more%20details.

Reply:
Question:
Is the `genome build` required to set up chromosome sizes in the plot?

Reply:
Correct, the genome_build/species is used to automatically let CIRCOS set up the chr sizes for the plot